### DISCUSSION TITLE
Add healing denial as a stat

### DISCUSSION THREAD

**Discussion by alyn509** [Heroes-Profile/heroesprofile#1042] [Ideas]
Ana or heroes which have healing reduction talents influence a lot the outcome of a match.

Healing denial should be a stat displayed and taken in consider in the match.

Had a match with Ana vs a team with Morales. My tank Garrosh if he got someone, no matter how deep would Morales go in to heal assist the thrown enemy, it could not be saved once I placed a biotic grenade. Ana gets no contribution as damage, but as healing denial can say that around 600 healing per ensured kill was there constantly.

**Comment by Zemill** [comment]
This data is not available in the replays unfortunately
